apps/test/unit/templates/sectionProgress/progressTables/ProgressTableStudentListTest.jsx

```diff
@@ -48,13 +55,27 @@ describe('ProgressTableStudentList', () => {
     const wrapper = setUp();
     const studentRows = wrapper.find(Virtualized.Body).props().rows;
     expect(studentRows).to.have.length(2);
-    expect(studentRows.includes(TEST_STUDENT_1)).to.be.true;
-    expect(studentRows.includes(TEST_STUDENT_2)).to.be.true;
+    expect(studentRows.includes(STUDENT_ROWS[0])).to.be.true;
+    expect(studentRows.includes(STUDENT_ROWS[1])).to.be.true;
   });
 
-  it('displays body with overflow scroll if needsGutter is true', () => {
-    const wrapper = setUp({needsGutter: true});
-    const virtualizedBodyComponent = wrapper.find(Virtualized.Body);
-    expect(virtualizedBodyComponent.props().style.overflowX).to.equal('scroll');
+  it('displays text cells if detail rows are passed in', () => {
+    // ProgressTableStudentName is a connected component so we need to stub
+    // the student name formatter to avoid setting up a store
+    sinon
+      .stub(ProgressTableStudentList.prototype, 'studentNameFormatter')
+      .callsFake(_ => <div />);
+
+    // reactabular initially only renders three rows, so we use a single
+    // student to avoid needing to workaround that.
+    const rows = [STUDENT_ROWS[0], ...DETAIL_ROWS];
+
+    // we need to use mount for the rows to actually be rendered
+    const wrapper = mount(
+      <ProgressTableStudentList {...DEFAULT_PROPS} rows={rows} />
+    );
+    expect(wrapper.find(ProgressTableTextLabelCell)).to.have.lengthOf(2);
+
+    sinon.restore();
```

Comment:
I thought that you needed to call `restore()` on the stub, not sinon (that's what it looks like in the docs). Are you sure this is the right syntax?

Reply:
`sinon` is an alias for the default [sandbox](https://sinonjs.org/releases/v9.2.4/sandbox/), so calling `sinon.restore()` restores the whole sandbox.